src/coredump/coredump-ratelimit.c

```diff
@@ -0,0 +1,170 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <sys/statvfs.h>
+#include "dirent-util.h"
+#include "fd-util.h"
+#include "fs-util.h"
+#include "string-util.h"
+#include "time-util.h"
+#include "user-util.h"
+#include "util.h"
+#include "coredump-ratelimit.h"
+#include "sd-id128.h"
+
+#define MAX_COMM_LENGTH 16
+
+static int comm_from_file_name(const char *file_name, char *comm) {
+        const char *p;
+        int i = 0;
+
+        /* check if the file is a core dump file */
+        p = startswith(file_name, "core.");
+        if (!p) {
+                /* check if the file is an inprogress core dump file */
+                p = startswith(file_name, ".#core.");
+                if(!p)
```

Comment:
style: space before bracket

Reply:
there are a few other instances of this - please fix them

Reply:
Addressed the comment. Please check once.